build/org.eclipse.birt.target/org.eclipse.birt.target.target

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target includeMode="feature" name="BIRT Photon target platform">
+	<locations>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/eclipse/updates/4.11/"/>
+			<unit id="org.eclipse.platform.sdk" version="0.0.0"/>
+			<unit id="org.eclipse.equinox.executable.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.cvs.feature.group" version="0.0.0"/>
+			<unit id="org.eclipse.rcp.configuration.feature.group" version="0.0.0"/>
+			<unit id="a.jre.javase" version="0.0.0"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/modeling/emf/emf/builds/milestone/latest"/>
```

Comment:
IMO it is better to reference the EMF that was a part of 4.11 to be consistent, and not the latest one.

Reply:
Which one was it? When I created this target platform file I only copied the repositories from the parent-pom.